daemon/daemon.go

```diff
@@ -528,6 +529,36 @@ func (daemon *Daemon) GetByName(name string) (*Container, error) {
 	return e, nil
 }
 
+// GetFilter returns a filters.Filter for a set of filters
+func (daemon *Daemon) GetFilter(filter filters.Args) *events.Filter {
```

Comment:
Ah right, I renamed this with the `s/EventFilter/Filter/`, but I should change this back.